### PR TITLE
Add FOV and resolution options, update for Hero9, and mention AUR package

### PR DESCRIPTION
**Pull request by NewoIsTaken**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -17,9 +17,9 @@ function yellow {
     printf "${YELLOW}$@${NC}\n"
 }
 
-curl -s https://raw.githubusercontent.com/jschmid1/gopro_as_webcam_on_linux/master/gopro -o /usr/local/sbin/gopro
+cp gopro /usr/local/sbin/
```
Comment (jschmid1) [inline install.sh:20]: Where do you get this from?

Comment (NewoIsTaken) [reply install.sh:20]: Oh, actually, that change can be reverted. Let me fix that.

Comment (NewoIsTaken) [reply install.sh]: All done!

File: gopro_webcam.service [hunk 2]
```diff
@@ -4,7 +4,7 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
-ExecStart=/usr/local/sbin/gopro webcam -p enx -a -n
+ExecStart=/usr/bin/gopro webcam -a -n
```
Comment (jschmid1) [inline gopro_webcam.service:7]: you start `/usr/bin/gopro` but copy the gopro script in `/usr/local/sbin/gopro`

Comment (NewoIsTaken) [reply gopro_webcam.service:7]: Yeah, that was a change for the AUR actually. I think would be best if we keep it in `/usr/local/sbin/`. Let me fix that.

File: gopro_webcam.service [hunk 3]
```diff
@@ -4,9 +4,9 @@ After=network-online.target
 Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
-ExecStart=/usr/local/sbin/gopro webcam -p enx -a -n
```
Comment (jschmid1) [inline gopro_webcam.service:7]: Could you elaborate on the reason you dropped this `-p enx` ?

Comment (NewoIsTaken) [reply gopro_webcam.service:7]: For my GoPro, the pattern is actually `enp` so I think specifying `-p enx` may cause some GoPros to not be detected.

Comment (jschmid1) [reply gopro_webcam.service:7]: Lets revert this to the original since its a per-user choice. If we change this we'd also need to change the README

Comment (jschmid1) [reply gopro_webcam.service:7]: oh, ok.. then lets keep it that way

**Comment by jschmid1** [comment]
Thanks for your contribution @NewoIsTaken

File: gopro_webcam.service (hunk 3 above)
Comment (jschmid1) [reply gopro_webcam.service:7]: https://github.com/jschmid1/gopro_as_webcam_on_linux/commit/325281b22c2f038470ca6651dcda79ffee1895a4

**Comment by NewoIsTaken** [comment]
Thank you for starting this project @jschmid1! Also, I believe there is another pull request that aims to do the same thing. I don't that one will be needed anymore.

**Comment by jschmid1** [comment]
>  I don't that one will be needed anymore. 

Yes. I was about to close it

File: gopro_webcam.service (hunk 3 above)
Comment (NewoIsTaken) [reply gopro_webcam.service:7]: @jschmid1 do you think it is necessary to put the actual service file in the README? I think we can just direct users to the actual file itself.

**Pull request merged**